tests/run-make/remap-path-prefix/rmake.rs

```diff
@@ -0,0 +1,89 @@
+// Generating metadata alongside remap-path-prefix would fail to actually remap the path
+// in the metadata. After this was fixed in #85344, this test checks that "auxiliary" is being
+// successfully remapped to "/the/aux" in the rmeta files.
+// See https://github.com/rust-lang/rust/pull/85344
+
+// FIXME(Oneirical): check if works without ignore-windows
```

Comment:
You can remove fixme, looks like it works :)

Reply:
Done!